canonical-web-and-design/ubuntu.com
# WIP - Update cve_list view so that it consumes security api

## Done

- Add function to api class for getting all cves
- Update cve_list view to api instead of querying the db directly 
- Refactor any broken filtering because of the above change
**NOTE: Refactoring is still in progress**

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/cves
    - Be sure to test on mobile, tablet and desktop screen sizes
- Make sure cves load successfully and that filtering works as expected 

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/11359
